emscripten/MarkerlessARJS_bindings.cpp

```diff
@@ -81,6 +81,40 @@ class ARPipeline_em {
   ARPipeline m_pipeline;
 };
 
+emscripten::val buildProjectionMatrix(const CameraCalibration& calibration, float nearPlane, float farPlane, int screen_width, int screen_height)
+{
+  Matrix44 projectionMatrix;
+
+  // Camera parameters
+  float f_x = calibration.fx(); // Focal length in x axis
+  float f_y = calibration.fy(); // Focal length in y axis (usually the same?)
+  float c_x = calibration.cx(); // Camera primary point x
+  float c_y = calibration.cy(); // Camera primary point y
+
+  projectionMatrix.data[0] = -2.0f * f_x / screen_width;
+  projectionMatrix.data[1] = 0.0f;
+  projectionMatrix.data[2] = 0.0f;
+  projectionMatrix.data[3] = 0.0f;
+
+  projectionMatrix.data[4] = 0.0f;
+  projectionMatrix.data[5] = 2.0f * f_y / screen_height;
+  projectionMatrix.data[6] = 0.0f;
+  projectionMatrix.data[7] = 0.0f;
+
+  projectionMatrix.data[8] = 2.0f * c_x / screen_width - 1.0f;
+  projectionMatrix.data[9] = 2.0f * c_y / screen_height - 1.0f;    
+  projectionMatrix.data[10] = -( farPlane + nearPlane) / ( farPlane - nearPlane );
+  projectionMatrix.data[11] = -1.0f;
+
+  projectionMatrix.data[12] = 0.0f;
+  projectionMatrix.data[13] = 0.0f;
+  projectionMatrix.data[14] = -2.0f * farPlane * nearPlane / ( farPlane - nearPlane );        
+  projectionMatrix.data[15] = 0.0f;
+
+  return emscripten::val{emscripten::typed_memory_view(16, &projectionMatrix.data[0])};
```

Comment:
this is not passing correctly the data array, i will change the approach.

Reply:
done with https://github.com/kalwalt/MarkerlessARJS/pull/5/commits/76d8af7f78103359aea232aba4cb56429aa86560